src/util.css

```diff
@@ -35,10 +35,26 @@
   justify-content: center;
 }
 
+.right {
```

Comment:
Use something like `.nav.right` instead of `.right`, as it might interfere with other classes.